tests/integration/test_skein_launcher.py

```diff
@@ -17,8 +17,9 @@
 
```

Comment:
Are these changes related to this issue here ?

Reply:
Tests were failing, maybe could have been FAPR